### PR TITLE
Database user routes

### LINKED ISSUE #98: Update user routes for database

### PR DESCRIPTION
**Pull request by SivanC**
Integrates the user information and platform routes with the MongoDB database Atlas instance. Closes #98.

### CODE REVIEW

File: back-end/users.js [hunk 1]
```diff
@@ -1,93 +1,97 @@
...
 const router = express.Router()
 
+const { User } = require('./models/User.js')
+
 // Multer handles file uploads
 // enable file uploads saved to disk in a directory named 'public/uploads'
 const storage = multer.diskStorage({
   destination: function (req, file, cb) {
-    cb(null, 'public/uploads')
+    cb(null, '../front-end/public/uploads')
```
Comment (byroncjy) [inline back-end/users.js:15]: Revert this back to backend, as we discussed
I'll configure frontend to access the backend path created

**Review by byroncjy — CHANGES_REQUESTED** [review]
Looks good, just need to remove check for authorized platforms because of custom platforms

File: back-end/models/User.js
[the remark this answers is not in the document]
Comment (SivanC) [reply back-end/models/User.js:1]: yup, will merge appropriately

File: back-end/platforms.js [hunk 2]
```diff
@@ -1,33 +1,31 @@
...
+router.put('/:id/platforms', async (req, res) => {
   try {
-    const userId = req.params.id
-    const platformInformation = req.body
+    const user = await User.findById(req.params.id)
+
+    const platforms = req.body.platforms
     // Authorized platform values
-    const authorizedPlatforms = [
+    const authorized = [
```
[the remark this answers is not in the document]
Comment (SivanC) [reply back-end/platforms.js:28]: Ok, sounds good

**Review by byroncjy — APPROVED** [review]
All good

**Pull request merged**